core/src/main/scala/com/eed3si9n/jarjarabrams/Main.scala

```diff
@@ -0,0 +1,81 @@
+package com.eed3si9n.jarjarabrams
+
+import com.eed3si9n.jarjar.{ MainUtil, RulesFileParser }
+import java.nio.file.Path;
+import scala.collection.JavaConverters._
+
+class Main {
+  def help(): Unit = Main.printHelp()
+
+  def process(rulesFile: Path, inJar: Path, outJar: Path): Unit = {
```

Comment:
This is the new entry point, which is used via reflection implemented in `MainUtil.runMain`.